manual/qmcpack_manual.tex

```diff
@@ -7,16 +8,16 @@
 \usepackage{algorithmic}
 \usepackage{algorithm}
 \usepackage{makeidx}
-\usepackage{ifpdf}
 \usepackage{fancyhdr}
 \usepackage{hyperref} %for urls
 %\usepackage{url}
 \usepackage{tabularx}
+\usepackage{xltabular}
```

Comment:
The xltabular package and the xltabular.sty file does not seem to be present in the Tex live 2015 distribution (Ubuntu 16.04)